config/express.js

```diff
@@ -97,6 +102,9 @@ module.exports = (app, config) => {
 
   app.use(constants.SITE_ROOT, express.static(`${config.root}/public`));
 
+  // metrics needs to be registered before routes wishing to have metrics generated
+  // see https://github.com/jochen-schweizer/express-prom-bundle#sample-uusage
```

Comment:
thought uusage would be a typo. Sadly not.

Reply:
Yeah, I don't know why it is `uUsage` rather than just `usage`. Any ideas?